[INFO   ] Logger: Record log in /storage/emulated/0/learn kivy/clock/.kivy/logs/kivy_22-12-24_51.txt
[INFO   ] Kivy: v2.0.0
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.9.7 (default, Oct  6 2021, 01:34:26) 
[GCC 11.1.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] Factory: 186 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 V@0530.0 (GIT@8382da08aa, I139f36ad46, 1645197494) (Date:02/18/22)'>
[INFO   ] GL: OpenGL vendor <b'Qualcomm'>
[INFO   ] GL: OpenGL renderer <b'Adreno (TM) 619'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] GL: NPOT texture support is available
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 730, in _apply_rule
[WARNING] stderr:     raise AttributeError(key)
[WARNING] stderr: AttributeError: presa
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/learn kivy/clock/main.py", line 30, in <module>
[WARNING] stderr:     Main().run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 949, in run
[WARNING] stderr:     self._run_prepare()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/app.py", line 919, in _run_prepare
[WARNING] stderr:     root = self.build()
[WARNING] stderr:   File "/storage/emulated/0/learn kivy/clock/main.py", line 27, in build
[WARNING] stderr:     return ClockApp()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/uix/boxlayout.py", line 145, in __init__
[WARNING] stderr:     super(BoxLayout, self).__init__(**kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/uix/layout.py", line 76, in __init__
[WARNING] stderr:     super(Layout, self).__init__(**kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/uix/widget.py", line 359, in __init__
[WARNING] stderr:     self.apply_class_lang_rules(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/uix/widget.py", line 463, in apply_class_lang_rules
[WARNING] stderr:     Builder.apply(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 541, in apply
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 737, in _apply_rule
[WARNING] stderr:     raise BuilderException(
[WARNING] stderr: kivy.lang.builder.BuilderException: Parser: File "/storage/emulated/0/learn kivy/clock/main.kv", line 105:
[WARNING] stderr: ...
[WARNING] stderr:     103:            text:'Stopwatch'  
[WARNING] stderr:     104:            on_presa:
[WARNING] stderr: >>  105:                root.ids.screen_manager.transition.direction ='down'
[WARNING] stderr:     106:                root.ids.screen_manager.current='stopwatch_screen'
[WARNING] stderr:     107:                                
[WARNING] stderr: ...
[WARNING] stderr: AttributeError: presa
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.9/site-packages/kivy/lang/builder.py", line 730, in _apply_rule
[WARNING] stderr:     raise AttributeError(key)
[WARNING] stderr: 
